src/main/resources/compile_schemas.sql

```diff
@@ -5,22 +5,56 @@ set serveroutput on
 set FEEDBACK OFF
 
 declare
+    -- The expected exception with the -20000 code is
+    -- "Unable to set values for index UTL_RECOMP_COMP_IDX1: does not exist or insufficient privileges".
+    -- If it was thrown, it will wait 10 seconds and attempt to compile again.
+    c_expected_error_code number := -20000;
```

Comment:
ORA-20000 is the generic exception, you should probably check error message or go deeper in the call stack